synapseutils/copy.py

```diff
@@ -439,7 +439,7 @@ def _updateInternalLinks(newWikis, wikiIdMap, entity, destinationId):
         newWikiId = wikiIdMap[oldWikiId]
         newWiki = newWikis[newWikiId]
         print("\tUpdating internal links for Page: %s\n" % newWikiId)
-        s = newWiki.markdown
+        s = newWiki.get("markdown")
```

Comment:
since we are under the assumption that `markdown` will always exist. we should use `['markdown']` instead of 'get()'
so that an error will be thrown if an invalid value is used.